edgar-feed/tests/health_check.rs

```diff
@@ -0,0 +1,21 @@
+#[tokio::test]
+async fn health_check_works() {
+    spawn_app();
+
+    let client = reqwest::Client::new();
+
+    let response = client
+        .get("http://127.0.0.1:8000/health")
+        .send()
+        .await
+        .expect("Failed to execute request.");
```

Comment:
yea, `expect` does this for you, it will panic and combine the error output with your custom message so in this case it'll be like

```
thread 'main' panicked at 'Failed to send request: reqwest::Error { kind: Request, url: "http://localhost:3010", source: hyper::Error(Io, Os { code: 110, kind: TimedOut, message: "Connection timed out" }) }', src/main.rs:4:10
```

or whatever